docs/machine-learning/resources/transforms.md

```diff
@@ -1,136 +1,185 @@
 ---
-title: Data transforms in ML.NET
-description: Explore the different data transforms supported in ML.NET.
+title: Machine learning data transforms - ML.NET
+description: Explore the feature engineering components supported in ML.NET.
 author: JRAlexander
-ms.date: 10/16/2018
+ms.custom: seodec18
+ms.date: 12/04/2018
 ---
-# Data transforms in ML.NET
-
-The following tables contain information about all of the data transforms supported in ML.NET (select the data transform type to navigate to the corresponding table):
-
-* [Categorical](#categorical)
-* [Combiners and segregators](#combiners-and-segregators)
-* [Feature selection](#feature-selection)
-* [Featurizers](#featurizers)
-* [Label parsing](#label-parsing)
-* [Missing Values](#missing-values)
-* [Normalization](#normalization)
-* [Row filters](#row-filters)
-* [Schema](#schema)
-* [Text processing and featurization](#text-processing-and-featurization)
-* [Miscellaneous](#miscellaneous)
+# Machine learning data transforms - ML.NET
```

Comment:
I find it a bit odd to have the - showing up in H1. I'd make the H1 different here and not use -

Maybe the original H1 was good and the non-branded terms are already added to the title metadata for SEO purposes.